src/mapping/attractor_mapping_recurrences.jl

```diff
@@ -71,6 +71,13 @@ want to search for attractors in a lower dimensional subspace.
   where a found attractor may intersect in the same cell with a new attractor the orbit
   traces (which leads to infinite resetting of all counters).
 
+### Grid irregularity specification
+* `density_matrix` while using regular grids of dimension 2, one can specify which regions 
```

Comment:
I am confused here. I thought we agreed that `density_matrix` is no longer a keyword we use. You need to rephrase the documentation strings and tell the users that they can provide grid three ways: 1) a tuple of AbstractRange, 2) a tuple of sorted AbstractVector, or 3) the output of the `subdivision_based_grid` function.